specification/communication/data-plane/CallingServer/preview/2021-06-15-preview/communicationservicescallingserver.json

```diff
@@ -0,0 +1,1467 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-06-15-preview",
+    "title": "Azure Communication Calling Server Service",
+    "description": "Azure Communication Calling Server Service"
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/calling/callConnections": {
+      "post": {
+        "tags": [
+          "CallConnection"
+        ],
+        "summary": "Create a new call.",
+        "description": "Create a new call.",
+        "operationId": "CallConnection_CreateCall",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "callRequest",
+            "in": "body",
+            "description": "Create call request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CreateCallRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
This seems like a long-running operation, you can annotate it as LRO e.g. [here](https://github.com/Azure/azure-rest-api-specs/blob/05306b337569ae87b2c56c92a0c4f3ac626ef723/specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json#L949)

Reply:
Could you point me to some doc on what the annotation "x-ms-long-running-operation" would do. The APIs for creating the call and hanging up the call rely on events coming back to the callbackUri in CreateCallRequest. The events give the callConnectionState to the application

Reply:
Per my understanding, these are not long running operations yet. The only way to get notified of operation completion is (currently) the notification callback.

In practical terms, it means that any generated library will change once the service adds support for polling for completion rather than the existing notification path. But since this is a preview, that is acceptable.

Reply:
Yes this is correct. We will revisit this post preview.